tests/testthat/test-miss.R

```diff
@@ -54,8 +56,8 @@ run.miss.test<-function(y){
   cat("Correct estimate =",theta,"with log-likelihood",llk,".\n")
   
   mplefit<-ergm(y~edges)
-  mple.theta.OK<-all.equal(theta,coef(mplefit),check.attributes=FALSE)
-  mple.llk.OK<-all.equal(llk,as.vector(logLik(mplefit)),check.attributes=FALSE)
+  mple.theta.OK<-all.equal(theta,coef(mplefit),ignore_attr=TRUE)
+  mple.llk.OK<-all.equal(llk,as.vector(logLik(mplefit)),ignore_attr=TRUE)
   cat("MPLE estimate =", coef(mplefit),"with log-likelihood",logLik(mplefit), if(isTRUE(mple.theta.OK)&&isTRUE(mple.llk.OK)) "OK.","\n")
```

Comment:
The tests currently fail with:

> Error (test-miss.R:80:3): directed network
> Error: Log-likelihood was not estimated for this fit. To get deviances, AIC, and/or BIC, use â*fit* <-logLik(*fit*, add=TRUE)â to add it to the object or rerun this function with eval.loglik=TRUE.
> Backtrace:
>   1. ergm:::run.miss.test(y) test-miss.R:80:2
>   8. ergm::logLik.ergm(mplefit)
>  10. base::with.default(...)
>  11. [ base::eval(...) ] with 1 more call
> 
> Error (test-miss.R:87:3): undirected Network
> Error: Log-likelihood was not estimated for this fit. To get deviances, AIC, and/or BIC, use â*fit* <-logLik(*fit*, add=TRUE)â to add it to the object or rerun this function with eval.loglik=TRUE.
> Backtrace:
>   1. ergm:::run.miss.test(y) test-miss.R:87:2
>   8. ergm::logLik.ergm(mplefit)
>  10. base::with.default(...)
>  11. [ base::eval(...) ] with 1 more call
> 
> Error (test-miss.R:94:3): Bipartite Undirected Network
> Error: Log-likelihood was not estimated for this fit. To get deviances, AIC, and/or BIC, use â*fit* <-logLik(*fit*, add=TRUE)â to add it to the object or rerun this function with eval.loglik=TRUE.
> Backtrace:
>   1. ergm:::run.miss.test(y) test-miss.R:94:2
>   8. ergm::logLik.ergm(mplefit)
>  10. base::with.default(...)
>  11. [ base::eval(...) ] with 1 more call

Reply:
For those that complain about likelihood not being evaluated, add `eval.loglik=TRUE` to the `ergm()` call, or, if it saves typing, set `opts <- options(ergm.eval.loglik=TRUE)` before and `opts <- options(opts)` after in order to undo the setting for the next file.

Reply:
fixed